src/Tizen.NUI/src/public/Size.cs

```diff
@@ -49,6 +57,14 @@ public Size() : this(Interop.Vector3.new_Vector3__SWIG_0(), true)
         /// <param name="x">The x (or width) component.</param>
         /// <param name="y">The y (or height) component.</param>
         /// <param name="z">The z (or depth) component(optional).</param>
+        /// <remarks>
+        /// Size2D and Size are implicitly converted to each other, so these are compatible and can be replaced without any type casting. <br />
+        /// For example, the followings are possible. <br />
+        /// view.Size2D = new Size(10.0f, 10.0f, 10.0f); // be aware that here the depth value(10.0f) will be lost. <br />
+        /// view.Size = new Size2D(10, 10); // be aware that here the depth value is 0.0f by default. <br />
+        /// view.MinimumSize = new Size(10, 10, 0); <br />
+        /// Size Tmp = view.MaximumSize; //here Tmp.Depth will be 0.0f. <br />
+        /// </remarks>
         /// <since_tizen> 5 </since_tizen>
         public Size(float x, float y, float z = 0.0f) : this(Interop.Vector3.new_Vector3__SWIG_1(x, y, z), true)
```

Comment:
Should really use width, height, depth for Size rather than x, y and z.

Reply:
Hello, I changed x,y,z into width,height.